cluster/gce/util.sh

```diff
@@ -696,6 +696,27 @@ function create-network() {
   fi
 }
 
+function delete-firewall-rules() {
+  for fw in $@; do
+    if [[ -n $(gcloud compute firewall-rules --project "${PROJECT}" describe "${fw}" --format='value(name)' 2>/dev/null || true) ]]; then
+      gcloud compute firewall-rules delete --project "${PROJECT}" --quiet "${fw}" &
+    fi
+  done
+  kube::util::wait-for-jobs || {
+    echo -e "${color_red}Failed to delete firewall rules.${color_norm}" >&2
+  }
+}
+
+function delete-network() {
+  if [[ -n $(gcloud compute networks --project "${PROJECT}" describe "${NETWORK}" --format='value(name)' 2>/dev/null || true) ]]; then
+    if ! gcloud compute networks delete --project "${PROJECT}" --quiet "${NETWORK}"; then
```

Comment:
It won't, typically, because it takes all the dependencies being gone before the `default` network can be deleted using `gcloud` (unlike the webconsole, gcloud is just trying to delete this single object). That's not usually going to be the case because `default` has other fw rules in it to start (the `default-allow` rules, etc.).